blockchain/gen_genesis.go

```diff
@@ -61,16 +59,6 @@ func (g *Genesis) UnmarshalJSON(input []byte) error {
 		GasUsed    *math.HexOrDecimal64                        `json:"gasUsed"`
 		ParentHash *common.Hash                                `json:"parentHash"`
 	}
-	printUnknownFields := func(input []byte) {
```

Comment:
@shiki-tak Please revive this logic. I know it's weird to modify an auto-generated file but we had no other workaround when we add this code.